install.sh

```diff
@@ -501,6 +518,24 @@ function configure_installation_variables() {
 				"Please, manually specify it with 'export SERVICE_CIDR=...' before executing again this script"
 	fi
 	info "[INSTALL] [CONFIGURE]" "Service CIDR: ${SERVICE_CIDR}"
+
+	# Set variables for kubernetes ingress
+	if [ -z "${LIQO_INGRESS_CLASS:-}" ] || [ -z "${LIQO_AUTHSERVER_ADDR:-}" ]; then
```

Comment:
I would avoid forcing to specify an ingress class. In case the parameter is not present, you could just omit the annotation and it would use the default ingress controller (e.g. in our cluster on-premise)